examples/crack_branching.cpp

```diff
@@ -62,26 +73,35 @@ int main( int argc, char* argv[] )
         Kokkos::Array<Kokkos::Array<double, 3>, 1> notch_positions = { p01 };
         CabanaPD::Prenotch<1> prenotch( v1, v2, notch_positions );
 
-        // Choose force model type.
+        // ====================================================
+        //                    Force model
+        // ====================================================
         using model_type =
             CabanaPD::ForceModel<CabanaPD::PMB, CabanaPD::Fracture>;
         model_type force_model( delta, K, G0 );
 
-        // Create particles from mesh.
+        // ====================================================
+        //                  Grid generation
+        // ====================================================
         // Does not set displacements, velocities, etc.
         auto particles = std::make_shared<
             CabanaPD::Particles<memory_space, typename model_type::base_model>>(
             exec_space(), low_corner, high_corner, num_cells, halo_width );
 
-        // Define particle initialization.
+        // ====================================================
+        //             Particle fields initialization
```

Comment:
This doesn't init, just gets the fields - these should move to where they're used below (either for BC or the actual particle init)

Reply:
Do you mean removing the section title and moving the following lines under other sections?
        auto rho = particles->sliceDensity();
        auto x = particles->sliceReferencePosition();
        auto u = particles->sliceDisplacement();
        auto v = particles->sliceVelocity();

Reply:
Yes